### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vueexcel**

### CODE REVIEW

File: components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="black wrapper" width="100%">
+    <div class="container px-sm-16 pl-7 px-0">
+      <v-row>
+        <v-col cols="12" sm="6" class="pl-sm-0">
+          <div>
+            <span class="text-uppercase white--text font-weight-bold"
+              >Invite your friends or post on social media</span
+            >
+            <div class="d-flex mt-5">
```
Comment (gulshanexcel) [inline components/Footer.vue:10]: all img tags looks same use loop instead

File: components/Footer.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+<script>
+export default {};
+</script>
+
+<style scoped>
+button {
+  outline: none;
+}
+.wrapper {
```
Comment (gulshanexcel) [inline components/Footer.vue]: remove this use vuetify ones instead

File: components/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+export default {};
+</script>
+
+<style scoped>
+button {
```
Comment (gulshanexcel) [inline components/Footer.vue]: there is no button on this page remove this if unused

File: components/Header.vue [hunk 4]
```diff
@@ -0,0 +1,113 @@
+<template>
+  <div>
+    <v-app-bar
+      class="black rounded-0 px-0 px-md-16 mx-auto"
+      height="100px"
+      width="100%"
+    >
+      <!-- <button @click="$router.go(-1)">
```
Comment (gulshanexcel) [inline components/Header.vue]: remove commented code before you push

File: components/Header.vue [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+      </v-app-bar-nav-icon>
+    </v-app-bar>
+    <v-navigation-drawer
+      v-model="drawer"
+      absolute
+      right
+      temporary
+      class="primary"
+    >
+      <v-list nav dense>
```
Comment (gulshanexcel) [inline components/Header.vue:44]: list items looks similar use loop instead

File: components/Header.vue [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+    return {
+      drawer: false,
+      group: null,
+    };
+  },
+  methods: {
+    goToHome() {
+      this.$router.push({
+        path: "/",
+        query: { ...this.$route.query },
```
Comment (gulshanexcel) [inline components/Header.vue]: why send query?

File: components/Header.vue [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+        path: "/login",
+        query: { ...this.$route.query },
+      });
+    },
+  },
+};
+</script>
+
+<style scoped>
+button {
```
Comment (gulshanexcel) [inline components/Header.vue]: look for vuetify utility class for it if been used everywhere then make it global

File: components/card.vue [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+      </div>
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+export default {
+  name: "card",
+  props: {
+    items: Array,
```
Comment (gulshanexcel) [inline components/card.vue]: always use default values in props

File: components/card.vue [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  methods: {
+    showDetails() {
+      this.$router.push({
+        path: "/:details",
+        query: { ...this.$route.query },
```
Comment (gulshanexcel) [inline components/card.vue]: again why query

File: components/card.vue [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+      });
+    },
+  },
+};
+</script>
+
+<style scoped>
+.card {
```
Comment (gulshanexcel) [inline components/card.vue]: can be replaced with utility classes

File: layouts/error.vue [hunk 11]
```diff
@@ -1,40 +1,38 @@
 <template>
```
Comment (gulshanexcel) [inline layouts/error.vue:1]: why send this file, I see all changes are due to code format, don't do that

File: pages/index.vue [hunk 12]
```diff
@@ -1,89 +1,175 @@
...
+}
+.browserWrapper:nth-child(odd) div .ImgWrapper {
+  text-align: left;
+  padding-right: 10px;
+}
+.browserWrapper:nth-child(even) div .ImgWrapper {
+  text-align: right;
+  padding-left: 10px;
+}
+.OnlineBrowseContainer img {
```
Comment (gulshanexcel) [inline pages/index.vue]: use css class name in cabab case

File: store/modules/user.module.js [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+    actions: {
+        async fetchUsers({ commit }) {
+            const data = await usersCollection.get()
+            data.docs.forEach((doc) => {
+                commit('userdetails', doc.data())
+            })
+        },
+    },
+    mutations: {
+        userdetails: (state, data) => {
```
Comment (gulshanexcel) [inline store/modules/user.module.js]: naming should be same throughout the app, userdetails should be USER_DETAILS

File: components/Footer.vue (hunk 1 above)
Comment (vueexcel) [reply components/Footer.vue:10]: done

File: components/Footer.vue (hunk 2 above)
Comment (vueexcel) [reply components/Footer.vue]: done

File: components/Footer.vue (hunk 3 above)
Comment (vueexcel) [reply components/Footer.vue]: done

File: components/Header.vue (hunk 4 above)
Comment (vueexcel) [reply components/Header.vue]: done

File: components/Header.vue (hunk 5 above)
Comment (vueexcel) [reply components/Header.vue:44]: done

Comment (vueexcel) [reply components/Header.vue:52]: done

File: components/card.vue (hunk 9 above)
Comment (vueexcel) [reply components/card.vue]: done

File: components/card.vue (hunk 10 above)
Comment (vueexcel) [reply components/card.vue]: done

File: layouts/error.vue (hunk 11 above)
Comment (vueexcel) [reply layouts/error.vue:1]: ok sir

File: pages/index.vue (hunk 12 above)
Comment (vueexcel) [reply pages/index.vue]: done

File: store/modules/user.module.js (hunk 13 above)
Comment (vueexcel) [reply store/modules/user.module.js]: yes sir, corrected

**Pull request merged**